templates/base.html

```diff
@@ -18,16 +18,23 @@
   {% block meta %}
     <meta property="og:type" content="website">
 
-    <title>{{ title() }}</title>
-    <meta property="og:title" content="{{ title() }}">
+    <title>Sponge NZ</title>
```

Comment:
`{{ title() }}` is a macro in jinja2. It's a convenient way to have unique titles for each page, like `Sponge -- Submissions`, `Sponge -- Archive`, etc (see the `{% block title %}` sections in the other html templates).

This change will make the title `Sponge NZ` on every page. It's cool if that's what you want, but in that case we should remove all the unique titles from the other templates.